samples/packaging/actively_supported/Gemfile

```diff
@@ -0,0 +1,6 @@
+# frozen_string_literal: true
+
+source "https://rubygems.org"
+
+gem "activesupport"
```

Comment:
Ha... gotta give the people what they want. Plus, I get my digs in with the message it presents in response to the gem's presence :)